src/renderStream/syncQueries.ts

```diff
@@ -1,6 +1,6 @@
 import {queries} from '@testing-library/dom'
 
-export {Queries} from '@testing-library/dom'
+export type {Queries} from '@testing-library/dom'
```

Comment:
Good catch!

Could be that eslint will expect this to be inside the `{}`, could you please check that?

Reply:
sure

Reply:
I checked, ESLint seems to prefer the `type` keyword to be outside the `{}`, but if you prefer it the other way around I can still change it.

Reply:
Nah, I'm fine with this, I just remember fighting ESLint over this yesterday :)
Thanks for the PR!